.vitepress/theme/components/AuthButtons/NavBarAuthButtons.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="container">
+        <a href="https://stackblitz.com/sign_in" class="link light">Sign in</a>
+        <a href="https://stackblitz.com/register" class="link accent">Register</a>
+    </div>
+</template>
+
+<style scoped lang="scss">
+@import './linkStyles.scss';
+
+.container {
+    margin-inline-start: 40px;
+    display: flex;
+    gap: 8px;
+    // this is when we hide top nav
+    @media screen and (max-width: 768px) {
+      margin-inline-start: 0px;
+    }
+    // we show the buttons inside the hamburger menu when on smaller screen
+    @media screen and (max-width: 400px) {
+      display: none;
+    }
+}
+
+.link {
+    @media screen and (max-width: 500px) {
+      height: 30px;
```

Comment:
```suggestion
      height: 40px;
```

Maybe just bump it up a bit on mobile.

Reply:
I assume for here right?
<img width="350" alt="Screenshot 2024-01-23 at 19 35 00" src="https://github.com/stackblitz/docs/assets/3702771/eef92215-9f3f-4017-814e-c7ccf0e0ced3">

Reply:
Correct 👍